hail/python/hailtop/batch/batch_pool_executor.py

```diff
@@ -139,7 +139,7 @@ def __init__(self, *,
         self.finished_future_count = 0
         self._shutdown = False
         version = sys.version_info
-        self.image = image or f'hailgenetics/python-dill:{version.major}.{version.minor}'
+        self.image = image or f'hailgenetics/python-dill:{version.major}.{version.minor}-slim'
```

Comment:
Can we add the slim image Docker files to the repository? Should we also assert here what the available version numbers are?

Reply:
None of these docker files are in the repository. I'll add them to docker, but its been a long outstanding task to have a solution for building and deploying docker images to docker hub; though that's now in question given the docker hub changes.

Reply:
I just build them locally and push them.